Health/onboarding_instructions.md

```diff
@@ -7,8 +7,11 @@ For on-boarding to Health(Tab), you would need to complete two steps
 
 ## Configure agent through ConfigMap
 1. Include the following section in ConfigMap yaml file
-```cmd:agent-settings: |-
-    [agent_settings.health_model]
+```
```

Comment:
I think we need to be more explicit here.
We need to mention, if they already have the cofigmap applied, they can append this to the file.(So that they dont override the existing configmap with default values)
Else, they can get the latest configmap file with default values and add this to the end.